dev-docs/bidders/adprime.md

```diff
@@ -2,8 +2,8 @@
 layout: bidder
 title: Adprime
 description: Prebid Adprime Bidder Adapter
-biddercode: adprime
-gdpr_supported: true
+biddercode: false
```

Comment:
@muuki88 - biddercode false? I think that will cause the Download page to not work